basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/ElytronOperations.java

```diff
@@ -214,7 +216,9 @@ public Address createSSLContext(Address sslContextAddress, Address keyManagerAdd
     public Address createCredentialStoreWithCredentialReferenceClearText(String credentialStoreName, String clearTextValue) throws IOException {
```

Comment:
Please avoid code duplication between all `createCredentialStoreWithCredentialReferenceClearText` methods. 
I know this PR doesn't bring this duplication but please fix it if you are already there.